src/Microsoft.Azure.WebJobs.Host/Scale/DefaultProcessMetricsProvider.cs

```diff
@@ -1,40 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-// This type was moved from https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script/Scale/DefaultProcessMetricsProvider.cs
-
-
-using System;
-using System.Diagnostics;
-
-namespace Microsoft.Azure.WebJobs.Host.Scale
-{
-    /// <summary>
-    /// Default implementation, that just delegates to the underlying Process.
-    /// </summary>
-    internal class DefaultProcessMetricsProvider : IProcessMetricsProvider
```

Comment:
This type/interface existed to make process stats mockable. I'm removing this and going deeper by making Process itself mockable, which was needed for some other tests I'm adding here around Process.HasExited.